components/card/card.test.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {mount, ReactWrapper} from 'enzyme';
+
+import Card from './card';
+import CardBody from './card_body';
+
+describe('components/card/card', () => {
+    const baseProps = {
+        expanded: false,
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = mount(
+            <Card {...baseProps}>
+                <Card.Header {...baseProps}>{'Header Test'}</Card.Header>
+                <Card.Body {...baseProps}>{'Body Test'}</Card.Body>
+            </Card>
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should have a height based on content when expanded', () => {
+        const wrapper: ReactWrapper<any, any, CardBody> = mount(
+            <Card.Body {...baseProps}>
+                {'Body Test'}
+                {'Slightly Larger Body Text'}
+            </Card.Body>
+        );
+
+        expect(wrapper.instance().card.current?.style.height).toBe('');
+
+        wrapper.setProps({expanded: true});
+        expect(wrapper.instance().card.current?.style.height).not.toBe('');
+
+        wrapper.setProps({expanded: false});
+        expect(wrapper.instance().card.current?.style.height).toBe('');
+    });
+});
```

Comment:
Snapshot tests are good but I'm not seeing any cypress tests. I'm ok if we're planning on adding those later but we should make sure we have cypress testing this new onboarding flow before we merge to master

Reply:
Yeah I have a ticket to add those tests near the end, I'm hesitant to do them now in case the flow needs to change or something else does. Ticket here: https://mattermost.atlassian.net/browse/MM-26477